packages/editor/src/components/EditorContainer.tsx

```diff
@@ -76,6 +78,18 @@ export const DockContainer = ({ children, id = 'editor-dock', dividerAlpha = 0 }
   )
 }
 
+const onEditorWarning = (warning) => {
+  console.warn(warning)
+  NotificationService.dispatchNotify(warning, {
+    variant: 'warning'
+  })
+
+  // popover design doesnt match the figma designs, we use notification for now
+  /*PopoverState.showPopupover(
+    <WarningDialog title={t('editor:warning')} description={warning || t('editor:warningMsg')} />
+  )*/
```

Comment:
design is using a notification
![image](https://github.com/user-attachments/assets/9def21ae-9b9b-4e69-b937-714dfa064d59)

hence, we can remove this comment

Reply:
i left it in in case we want to build a customized modal around it later, i would suggest keeping it in, it will reduce dev effort later